<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_10" columnstretch="0,0,0">
    <item row="1" column="0">
     <widget class="QTabWidget" name="code_chart">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="teb_daily">
       <attribute name="title">
        <string>일봉</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="PlotWidget" name="day_chart"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_weekly">
       <attribute name="title">
        <string>분봉</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="weekly_chart"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="market_chart">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="kospi">
       <attribute name="title">
        <string>kospi</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="PlotWidget" name="kospichart"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="kosdaq">
       <attribute name="title">
        <string>kosdaq</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="kosdaqchart"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="future">
       <attribute name="title">
        <string>선물</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="futurechart"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QWidget" name="hoga_order" native="true">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>호가주문</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="1">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_14" rowstretch="80,50,800">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QTableWidget" name="table_hoga1">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustIgnored</enum>
                </property>
                <property name="autoScroll">
                 <bool>true</bool>
                </property>
                <property name="autoScrollMargin">
                 <number>16</number>
                </property>
                <property name="rowCount">
                 <number>1</number>
                </property>
                <property name="columnCount">
                 <number>7</number>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>85</number>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>30</number>
                </attribute>
                <row/>
                <column>
                 <property name="text">
                  <string>호가종목</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>매입가</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>현재가</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>수익률</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>평가손익</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>매입금액</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>평가금액</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QWidget" name="widget_2" native="true">
                <widget class="QRadioButton" name="radioButton">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>10</y>
                   <width>61</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>50%</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="geometry">
                  <rect>
                   <x>80</x>
                   <y>10</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>100%</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="geometry">
                  <rect>
                   <x>150</x>
                   <y>10</y>
                   <width>51</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>1주</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="geometry">
                  <rect>
                   <x>210</x>
                   <y>10</y>
                   <width>91</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>백만원</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="table_hoga2">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="rowCount">
                 <number>22</number>
                </property>
                <property name="columnCount">
                 <number>7</number>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>85</number>
                </attribute>
                <attribute name="horizontalHeaderHighlightSections">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>32</number>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string>매도주문</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>체결수량</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>직전대비</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>잔량</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>호가</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>증감률</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>매수주문</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>종목뉴스</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="100,200">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="control">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>수신로그</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>모드설정</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="gwansim">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_gwansim">
          <attribute name="title">
           <string>관심종목</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="table_gwansim">
             <property name="cursor" stdset="0">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="rowCount">
              <number>20</number>
             </property>
             <property name="columnCount">
              <number>7</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>82</number>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>32</number>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>    종목명  </string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>현재가</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>대비</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>등락율</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>거래량</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>거래대금</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>체결강도</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_account">
          <attribute name="title">
           <string>계좌잔고</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="table_account">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>20</number>
             </property>
             <property name="columnCount">
              <number>7</number>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>90</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>종목명</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>보유수량</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>매입가</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>현재가</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>수익률(%)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>평가손익</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>평가금액</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_acc_p">
          <attribute name="title">
           <string>계좌평가</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QTableWidget" name="table_acc_eva">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>16</number>
             </property>
             <property name="columnCount">
              <number>6</number>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>95</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>계좌번호</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>추정예탁자산</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>수익률(%)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>총평가손익</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>총매입금액</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>총평가금액</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
